modules/libcreate/3.1.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,140 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,137 @@
++load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
++
++licenses(["notice"])  # BSD-3-Clause
++
++###############
++#  Libraries  #
++###############
++
++cc_library(
++    name = "libcreate",
```

Comment:
Could you call this target `create` and add an alias named `libcreate`? Otherwise the artifact will be called `liblibcreate.so`, which is confusing.